### DISCUSSION TITLE
About the application scenarios supported by the program

### DISCUSSION THREAD

**Discussion by KOOKOKOK** [yoshitomo-matsubara/torchdistill#463] [Q&A]
Hi, I would like to know if all the reproduced paper methods in this project support semantic segmentation, categorization, and target detection, or is it just the original paper that corresponds to the work. For example, if the original paper is doing knowledge distillation about target detection, then the pages in the project only support target detection.

**Comment by yoshitomo-matsubara** [comment]
Hi @KOOKOKOK ,

Reimplemented methods in this framework can work for different tasks unless the original methods are designed for specific tasks.
For instance, you can use Hinton et al.'s standard KD for image classification, text classification, etc.
